Firebase/Auth/Source/AuthProviders/Phone/FIRPhoneAuthProvider.m

```diff
@@ -76,21 +124,76 @@ - (void)verifyPhoneNumber:(NSString *)phoneNumber
         });
       }
     };
+    [self internalVerifyPhoneNumber:phoneNumber completion:callBackOnMainThread];
```

Comment:
Now you don't need the `callBackOnMainThread` variable but just using trailing block for better readability.

Reply:
Agreed, removed.